tests/test_animesvostfr.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+python3 -m pip install ../.
+anime -ll DEBUG dl --provider animesvostfr "youjo senki"
```

Comment:
Tests for sites should ideally be in test_sites and actually test things instead of just debugging.

I recommend ommiting this

Reply:
OK, maybe i could put *.sh in .gitignore ?